cmd/zetae2e/stress.go

```diff
@@ -88,27 +89,15 @@ func StressTest(cmd *cobra.Command, _ []string) {
 	cosmosConf.Seal()
 
 	// initialize E2E tests config
-	conf, err := local.GetConfig(cmd)
-	if err != nil {
-		panic(err)
-	}
+	conf := must(local.GetConfig(cmd))
 
 	// Initialize clients ----------------------------------------------------------------
-	evmClient, err := ethclient.Dial(conf.RPCs.EVM)
-	if err != nil {
-		panic(err)
-	}
+	evmClient := must(ethclient.Dial(conf.RPCs.EVM))
+	bal := must(evmClient.BalanceAt(context.TODO(), local.DeployerAddress, nil))
```

Comment:
Honestly I find this harder to read than the original. I would personally perfer
```
evmClient, err := ethclient.Dial(conf.RPCs.EVM)
ensure(err)
```